src/lib/components/works_for_you/notification.tsx

```diff
@@ -13,7 +13,9 @@ interface Props extends RelayProps {}
 
 export class Notification extends React.Component<Props, any> {
   handleArtistTap() {
-    const artistHref = this.props.notification.artworks[0].artists[0].href
+    // Special notifications will pass down an artistHref. Otherwise, grab it from the artworks
```

Comment:
It just refers to this special-case situation where we make a notification from an artist's recently published works instead of through our usual notifications api.